src/views/product-root-docs-path-landing/index.tsx

```diff
@@ -20,7 +20,20 @@ const ProductRootDocsPathLanding = ({
   const showProductDocsSearch = __config.flags.enable_product_docs_search
 
   let mdxSlot: ReactElement
-  if (mdxSource) {
+  /**
+   * TODO: cloud.hashicorp.com/docs redirects to cloud.hashicorp.com/docs/hcp,
+   * and the content for the former URL is empty (just a heading).
+   * This is a workaround to prevent the empty content from being rendered.
+   * https://github.com/hashicorp/cloud.hashicorp.com/blob/main/content/docs/index.mdx
+   *
+   * Related redirects-in-dev-dot issues:
```

Comment:
Have noted in PR description as well, but for now, redirects authored in cloud.hashicorp.com are not being applied. Intent is to defer this work outside this PR.